src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResourcePagingTest.kt

```diff
@@ -224,4 +224,15 @@ class CourtResourcePagingTest : IntegrationTest() {
       .jsonPath("$.content[0].buildings[0].contacts[0].type").isEqualTo("TEL")
       .jsonPath("$.content[0].buildings[0].contacts[0].detail").isEqualTo("01545 570886")
   }
+
+  @Test
+  fun `a search finding multiple buildings and contacts should only return the court once`() {
```

Comment:
since two separate queries under the covers should have really got two separate tests for the different code paths

Reply:
It's just a single query under the covers that join courts, buildings and contacts.

But clearly the test name and comment have confused you so I'll have a think about how to better express my intentions with this test.

Reply:
my point was that you changed the `CourtRepository` in two separate places to add in `distinct`, but there was only one test addition instead of two, which therefore means that only one of the `CourtRepository` changes is untested?

Reply:
Aha!  Yes... the first query currently cannot return duplicates value and so is covered by regression tests.  I added `distinct` just for documentation to indicate that it too will never return duplicates.  But maybe it shouldn't be there as it's redundant. :thinking: